generators/app/templates/generators/client/_index.js

```diff
@@ -63,19 +63,19 @@ module.exports = class extends ClientGenerator {
 
     get prompting() {
         // The prompting phase is being overriden so that we can ask our own questions
-        return {
-            askForClient: prompts.askForClient,
-            askForClientSideOpts: prompts.askForClientSideOpts,
+        // return {
```

Comment:
I commented all this part, as VueJS is not implemented yet :laughing: 
Otherwise, Travis builds will failed

Reply:
but this was to test if overriding works and the new file was ceated

Reply:
Yes you're right, but the default generated blueprint must work. That's why I kept the code in comments, instead of delete it. So we can fix it

Reply:
oh you mean the generated app should work. Ok I get it